apps/Hyper/description

```diff
@@ -0,0 +1,4 @@
+A modern electron terminal
```

Comment:
Try to be more descriptive and explain why it's better than alacritty, eDEX-UI, tabby, lxterminal, guake, etc.